### ISSUE TITLE
WorldEdit prevents SpongeForge 1.16.5 from loading

### ISSUE THREAD

**Issue by reiyooki** [EngineHub/WorldEdit#1899]
### WorldEdit Version

7.2.5

### Platform Version

forge-1.16.5-36.2.4, spongeforge-1.16.5-36.2.1-8.0.0-RC900

### Confirmations

- [X] I am using the most recent Minecraft release.
- [X] I am using a version of WorldEdit compatible with my Minecraft version.
- [X] I am using the latest or recommended version of my platform software.
- [X] I am NOT using a hybrid server, e.g. a server that combines Bukkit and Forge. Examples include Arclight, Mohist, and Cardboard.
- [X] I am NOT using a fork of WorldEdit, such as FastAsyncWorldEdit (FAWE) or AsyncWorldEdit (AWE)

### Bug Description

WorldEdit refuses to load, and prevents server startup. The full crash report is linked [in this gist](https://gist.github.com/reiyooki/0917ea623bc2393a261833bc1a573924)

### Expected Behavior

Expected the server to load, as WorldEdit loads perfectly fine when the SpongeForge RC is removed.

### Reproduction Steps

1. Place both WorldEdit 7.2.5 & SpongeForge 1.16.5 RC-900 in mods directory
2. Start server

### Anything Else?

The SpongeForge dev build used to test this is [Here](https://repo.spongepowered.org/#browse/search=attributes.maven2.artifactId%3Dspongeforge%20AND%20version%3D1.16.5*:85135652a978be9a13111c8a78a8f9a0:a81fe9f475e8d3b9e97d9a22a73a78cd)

**Comment by wizjany** [comment]
would have to remove/change the dep in mods.toml. iirc the perms hook is commented out since there was no sponge to build on at the time we updated forge. also not sure that we're going back to 1.16.5 anymore, at least in terms of releases. dunno how versioning would work out.

**Comment by reiyooki** [comment]
It appears that just removing the dep in mods.toml does indeed let WorldEdit (and the server) load, but it throws a stacktrace on startup. Looks like it's to do with Sponge's command handler.

```
[17Sep2021 09:29:17.062] [Server thread/INFO] [com.sk89q.worldedit.extension.platform.PlatformCommandManager/]: Registering commands with com.sk89q.worldedit.forge.ForgePlatform
[17Sep2021 09:29:17.065] [Server thread/ERROR] [com.sk89q.worldedit.util.eventbus.EventBus/]: Could not dispatch event: com.sk89q.worldedit.event.platform.PlatformReadyEvent@35c046b6 to handler EventHandler{priority=NORMAL}
java.lang.reflect.InvocationTargetException: null
	at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:75) ~[worldedit:?]
	at com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java:193) ~[worldedit:?]
	at com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java:181) ~[worldedit:?]
	at com.sk89q.worldedit.forge.ForgeWorldEdit.serverStarted(ForgeWorldEdit.java:214) ~[worldedit:?]
	at net.minecraftforge.eventbus.ASMEventHandler_9_ForgeWorldEdit_serverStarted_FMLServerStartedEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-4.0.0.jar:?]
	at org.spongepowered.forge.launch.event.SpongeEventBus.post(SpongeEventBus.java:120) ~[spongeforge:1.16.5-36.2.1-8.0.0-RC900]
	at org.spongepowered.forge.launch.event.ForgeEventManager.post(ForgeEventManager.java:125) ~[spongeforge:1.16.5-36.2.1-8.0.0-RC900]
	at org.spongepowered.forge.launch.event.ForgeEventManager.post(ForgeEventManager.java:111) ~[spongeforge:1.16.5-36.2.1-8.0.0-RC900]
	at net.minecraftforge.fml.server.ServerLifecycleHooks.handleServerStarted(ServerLifecycleHooks.java:106) ~[forge:?]
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:622) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_301]
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_301]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_301]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_301]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_301]
	at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[worldedit:?]
	at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[worldedit:?]
	... 12 more
Caused by: java.lang.IllegalStateException: Cannot register command without knowing its origin.
	at org.spongepowered.common.command.registrar.BrigadierCommandRegistrar.lambda$register$0(BrigadierCommandRegistrar.java:96) ~[spongeforge:1.16.5-36.2.1-8.0.0-RC900]
	at java.util.Optional.orElseThrow(Unknown Source) ~[?:1.8.0_301]
	at org.spongepowered.common.command.registrar.BrigadierCommandRegistrar.register(BrigadierCommandRegistrar.java:96) ~[spongeforge:1.16.5-36.2.1-8.0.0-RC900]
	at org.spongepowered.common.command.brigadier.dispatcher.DelegatingCommandDispatcher.register(DelegatingCommandDispatcher.java:55) ~[spongeforge:1.16.5-36.2.1-8.0.0-RC900]
	at com.sk89q.worldedit.forge.CommandWrapper.register(CommandWrapper.java:67) ~[worldedit:?]
	at com.sk89q.worldedit.forge.ForgePlatform.registerCommands(ForgePlatform.java:168) ~[worldedit:?]
	at com.sk89q.worldedit.extension.platform.PlatformCommandManager.registerCommandsWith(PlatformCommandManager.java:449) ~[worldedit:?]
	at com.sk89q.worldedit.extension.platform.Capability$3.initialize(Capability.java:64) ~[worldedit:?]
	at com.sk89q.worldedit.extension.platform.PlatformManager.choosePreferred(PlatformManager.java:187) ~[worldedit:?]
	at com.sk89q.worldedit.extension.platform.PlatformManager.handlePlatformReady(PlatformManager.java:300) ~[worldedit:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_301]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_301]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_301]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_301]
	at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58) ~[worldedit:?]
	at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73) ~[worldedit:?]
	... 12 more
```

**Comment by wizjany** [comment]
that one's not on us

**Comment by reiyooki** [comment]
Noted, will report that one to Sponge team.

**Comment by AngerminecraftYT** [comment]
Failure message: Mod worldedit only supports sponge 1.13
		Currently, sponge is 8.0.0

that was in the log!
please try with 1.13

**Comment by octylFractal** [comment]
This issue was filed correctly. Please don't comment on things with unhelpful information.

**Comment by octylFractal** [comment]
Fixed by #1900.
